pkg/api/mlflow/common/helpers_test.go

```diff
@@ -0,0 +1,42 @@
+package common
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// TestGetContentType the TestGetContentType private helpers
+func TestGetContentType(t *testing.T) {
+	testCases := []struct {
```

Comment:
sometimes we use `testCases` sometimes `testData`, can you please unity it? for me `tests` just works really well.

Reply:
done